doc/source/walkthrough.rst

```diff
@@ -3,10 +3,20 @@ Ray Core Walkthrough
 
 This walkthrough will overview the core concepts of Ray:
 
-1. Starting Ray
-2. Using remote functions (tasks) [``ray.remote``]
-3. Fetching results (object IDs) [``ray.put``, ``ray.get``, ``ray.wait``]
-4. Using remote classes (actors) [``ray.remote``]
+.. tabs::
```

Comment:
Overall this is great work - good to see the tabs and think they will add a lot to the community.

With respect to some guiding principles having worked on Spark is that targeting having the fewest number of tab groups. It becomes very complicated to maintain over time and to enable other contributors. 

The one major request I would have is that It'd be good to minimize the contents of each tab group to only include the minimum amount of information as possible - even if that involves a little reorganization of the information that we currently have.